rest/data/commit_queue.go

```diff
@@ -165,6 +166,7 @@ func writePatchInfo(patchDoc *patch.Patch, patchSummaries []thirdparty.Summary,
 	return nil
 }
 
+// if bool is true, move the commit queue item to be processed next.
```

Comment:
nit: comments should start with function name (can we just shore this comment up though, since that just explains enqueueNext)